src/components/input-field/partial-styles/cropped-label-hack.scss

```diff
@@ -0,0 +1,19 @@
+// Some font size applied to `label--float-above` causes the labels to get cut off
+// when an empty field gets focused
+
+.mdc-text-field--outlined.mdc-notched-outline--upgraded,
+.mdc-text-field--outlined .mdc-notched-outline--upgraded {
+    .mdc-floating-label--float-above {
+        //font-size: 1rem; This is what we get from MD now, which causes the miscalculations
+        font-size: 0.875rem;
+        // of course this is `14px` and the other one is `16px`.
+        // Unfortunately MD scales the floating label down, by applying a
+        // `transform` & `scale(0.75)` which is probably why they had to increase
+        // the font-size, to make it more readable.
+        // This is why I don't like this hack.
```

Comment:
> which is probably why they had to increase the font-size

Who are "they"?

I think we increased the font-size, because I'm reasonably sure this bug doesn't appear in MDC's examples.

Reply:
I don't find anywhere in the code that we deliberately change the font size for floating labels of input fields. This must be Material Design's originally.

![Screen Shot 2021-05-17 at 15 26 49 (2)](https://user-images.githubusercontent.com/35954987/118496710-aa5f2580-b724-11eb-9a96-bdbaf2415713.png)

Reply:
Could it not be due to our px to rem conversions?

Reply:
I don't know. Unless that conversion code is also rounding up numbers (which I don't think it does)?

Reply:
or is it JavaScript that is having mathematical problems (e.g. with decimals as Adrian was saying this morning?)

Reply:
The pxToRem is done completely by Sass while compiling the scss to css. While I can't completely rule it out, I think it's unlikely to be caused by the conversion itself. I think it's more plausible that it could be caused by us using rem rather than px (that is, it wouldn't matter if we use the `pxToRem()` function or just defined values in `rem`, but it _would_ matter if we used `px` _instead_ of `rem`).

I still have this distinct memory that we somehow increased the font size for floating labels back in the day, but perhaps that was for Angular Material, not for MDC… 🤔

Reply:
> or is it JavaScript that is having mathematical problems (e.g. with decimals as Adrian was saying this morning?) So while converting, it messes up this number?

I think that's very unlikely, because the error will only ever be fractional, so it would have to be multiplied by at least a factor of 10, but more likely at least 100, to even cause an error of a single pixel…

Reply:
I wasn't talking about our `pxToRem` function. I thought there is some script that automatically converts `px` values that are inside MD's original css files which we import and load. Because I don't find anything in our scss files.

Reply:
Ah. Yes, there is a script that converts MDC's px-values to rem-values. There could be something going wrong there, true… 🤔

The script is here: https://github.com/Lundalogik/lime-material-scripts/blob/master/convert.js
Review is welcome! 😊

Reply:
@jensgustafsson, he's talking to you 😜

Reply:
I guess we need to compare the lines of code after/before converting because that script doesn't tell me too much 😛🙈

Reply:
But I'm fine with your change @Kiarokh. Not sure what @adrianschmidt thinks though

Reply:
Yeah, I am too! 👍